modules/oadp-release-notes-1-2-0.adoc

```diff
@@ -34,5 +34,12 @@ For a complete list of all issues resolved in this release, see the list of link
 [id="known-issues-1-2-0_{context}"]
 == Known issues
 
-This release does not have any known issues.
+This release has the following known issues:
 
+.An incorrect hostname can be created when changing a hostname in a generated route.
+
+By default, OpenShift cluster makes sure that the openshift.io/host.generated: 'true' annotation is turned on and fills in the field for both the routes that are generated and those that are not generated.
```

Comment:
@anarnold97  Sorry for this review out of the label process but I got the notification and I happened to check that the backticks are missing here. The annotation should be in backticks: `openshift.io/host.generated: 'true'`

Reply:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references